azurelinuxagent/ga/update.py

```diff
@@ -94,26 +95,32 @@ def run_latest(self):
         """
         latest_agent = self.get_latest_agent()
         if latest_agent == None:
-            agent_bin = sys.argv[0]
+            agent_cmd = "python -u {0} -run-exthandlers".format(sys.argv[0])
```

Comment:
...well apparently `-u` won't work in the shebang. that didn't take me long to realize (unix passes "-u filename.py" as a single argument to python).. anyways my python3 to python2 downgrading comment should still be valid.